spec/unit/lib/cloud_controller/permissions_spec.rb

```diff
@@ -752,30 +735,30 @@ module VCAP::CloudController
         it 'returns true for space developer' do
           org.add_user(user)
           space.add_developer(user)
-          expect(permissions.can_manage_apps_in_active_space?(space_guid)).to be true
-        end
-
-        it 'returns true for space supporter' do
-          org.add_user(user)
-          space.add_supporter(user)
-          expect(permissions.can_manage_apps_in_active_space?(space_guid)).to be true
+          expect(permissions.can_write_to_active_space?(space.id)).to be true
         end
 
         it 'returns false for space manager' do
           org.add_user(user)
           space.add_manager(user)
-          expect(permissions.can_manage_apps_in_active_space?(space_guid)).to be false
+          expect(permissions.can_write_to_active_space?(space.id)).to be false
```

Comment:
Was `expect(permissions.can_manage_apps_in_active_space?(space_guid)).to be false` before.